recipe/meta.yaml

```diff
@@ -10,37 +10,29 @@ source:
   sha256: 702f1dffc5799e47d879383bbf66bf4f1baa92e40e6241dc9e59845b6ff4df2c
 
 build:
-  number: 1
+  number: 2
   script:
     - "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   host:
-    - future
+    # - future
     - python
     - pip
-    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
-    - numpy  # [not (win and py2k)]
-    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
-    - scipy  # [not (win and py2k)]
-    - libblas ==3.8.0 8_mkl  # [win and py2k]
-    - libcblas ==3.8.0 8_mkl  # [win and py2k]
-    - liblapack ==3.8.0 8_mkl  # [win and py2k]
+    # - numpy
+    # - scipy
   run:
     - future
     - python
-    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
-    - numpy  # [not (win and py2k)]
-    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
-    - scipy  # [not (win and py2k)]
+    - {{ pin_compatible('numpy') }}
+    - scipy
     - gmsh <4.0   # [not (win and (py2k or x86))]
-    - mpich       # [not win]
-    - mpi4py      # [not win]
-    - petsc4py    # [not win]
-    - pysparse    # [not py3k]
-    - pytrilinos ==12.10.1 py27h7127e17_2  # [not win and not py3k]
```

Comment:
Earlier builds of pytrilinos are broken, but the solver doesn't find build 2

Reply:
haha, ok lets work through it slowly. I can't actually remove packages for you, so i figured I would help build a new one ;)

Reply:
Got it. Thank you very much for your help.